src/functions/sendWeeklyMessage/handler.ts

```diff
@@ -26,7 +31,7 @@ const sendWeeklyMessage: ValidatedEventAPIGatewayProxyEvent<typeof schema> = asy
 
     const weeklyCombinedData = TimeBankUtilities.combineWeeklyData(timeEntries, slackUsers);
 
-    SlackApiUtilities.postWeeklyMessage(weeklyCombinedData, weekStartDate.toISODate(), weekEndDate.toISODate());
+    SlackApiUtilities.postWeeklyMessage(weeklyCombinedData, weekStartDate.toISODate(), weekEndDate.toISODate(), timeRegistrations, yesterdaysAndTodaysDates);
```

Comment:
5 parameters to single function is starting to be a bit too much. Is all data really needed in the slack utilities class if the main purpose is just to send slack messages?

Reply:
Done